app/models/katello/concerns/subscription_aspect_host_extensions.rb

```diff
@@ -5,7 +5,18 @@ module SubscriptionAspectHostExtensions
       include ForemanTasks::Concerns::ActionTriggering
 
       included do
+        SUBSCRIPTION_STATUS_MAP = {
+          :valid => Katello::SubscriptionStatus::FULL_COMPLIANCE,
```

Comment:
At the source level does this matter? Candlepin itself uses the word "compliant" both in the code and the json results. I believe that displaying "entitled" to the user is the proper terminology.